ktor-client/ktor-client-cio/common/src/io/ktor/client/engine/cio/EngineTasks.kt

```diff
@@ -15,7 +15,9 @@ internal data class RequestTask(
     val request: HttpRequestData,
     val response: CompletableDeferred<HttpResponseData>,
     val context: CoroutineContext
-)
+) {
+    override fun toString(): String = hashCode().toString()
```

Comment:
Is it leftover from debugging or by design? I think it's useful to see at least a request url.

Reply:
Fixed! Thanks!